node-metrics/src/service/data_state/node_identity.rs

```diff
@@ -0,0 +1,248 @@
+use super::LocationDetails;
+use espresso_types::FeeAccount;
+use hotshot_types::signature_key::BLSPubKey;
+use serde::{Deserialize, Serialize};
+use surf_disco::Url;
+
+/// [NodeIdentity] represents the identity of the node that is participating
+/// in the network.
+#[derive(Clone, PartialEq, Debug, Serialize, Deserialize)]
+pub struct NodeIdentity {
+    pub(crate) public_key: BLSPubKey,
+    pub(crate) name: Option<String>,
+    pub(crate) wallet_address: Option<FeeAccount>,
```

Comment:
Hmm.. My thoughts / understanding on this would be that this might point to the wallet address of the `builder` that the `node` might eventually rerpesent.  Perhaps this is mean to be the wallet address of the sequencer who may be getting rewards for their efforts in the future.  What do you think?

Reply:
Hmm. Builders and consensus nodes are totally separate logical roles, even if they happen to be run by the same operator (which isn't even the case necessarily). So I don't think it makes sense to couple builder wallet here. Reward address might make more sense, but we don't have a design for validator rewards yet, so I feel this might be something better to leave out for now and add in later when we have more clarity

Reply:
Resolved in 0ef688df10e905f97c4f0ecb6c2d54b2d7014f7d.